src/reducer.js

```diff
@@ -96,6 +96,15 @@ const initialState = {
   isShowingPinScreen: false,
   walletStartState: WALLET_STATUS.LOADING,
   pushNotification: {
+    /**
+     * optInQuestionDismissed {boolean} if user has dismissed the push notification modal
+     * this is used to show the modal only the first time user opens the app
+     * after the modal is dismissed, it will not be shown again
+     * until the user clears the app data
+     * (this is done to avoid showing the modal to users that have already
+     * enabled push notification)
+     */
+    optInQuestionDismissed: true,
```

Comment:
Shouldn't this be initialized with `false`?

Reply:
No. It is initialized as dismissed, and in the push initialization verifies if there is a dismiss stored, if not, it forces it to appear.

Reply:
Sounds good. I will work on it.